File: cases/taint/javascript/local-overwrite-positive/LocalOverwritePositive.js
function dfb_source() { // DFB-SOURCE: local-overwrite-input
  return "tainted";
}

function dfb_sink(value) {} // DFB-SINK: local-overwrite-sink

function run() {
  let value = dfb_source();
  value = value; // DFB-WITNESS: local-overwrite-preserved
  dfb_sink(value);
}

File: cases/taint/scala/local-overwrite-positive/LocalOverwritePositive.scala
package dataflowbench

object LocalOverwritePositive {
  def dfb_source(): Int = { // DFB-SOURCE: local-overwrite-input
    1
  }

  def dfb_sink(value: Int): Unit = {} // DFB-SINK: local-overwrite-sink

  def run(): Unit = {
    var value = dfb_source()
    value = value // DFB-WITNESS: local-overwrite-preserved
    dfb_sink(value)
  }
}
